src/plugins/intel_npu/tests/functional/behavior/work_with_devices.cpp

```diff
@@ -10,7 +10,7 @@
 namespace {
 
 const std::vector<ov::AnyMap> configs = {
-    {{ov::log::level(ov::log::Level::ERR)}},
+    {{ov::log::level(ov::log::Level::DEBUG), ov::intel_npu::bypass_umd_caching(true)}},
```

Comment:
```suggestion
    {{ov::intel_npu::bypass_umd_caching(true)}},
```

Please remove logs from tests